subsys/net/lib/lwm2m_client_utils/lwm2m/lwm2m_lte_notification.c

```diff
@@ -191,6 +191,17 @@ void lte_notify_handler(const struct lte_lc_evt *const evt)
 		lwm2m_lte_reg_handler_notify(evt->nw_reg_status);
 		break;
 #endif /* CONFIG_LWM2M_CLIENT_UTILS_NWK_REG_NOTIFICATION */
+
+#if defined(CONFIG_LTE_LC_TAU_PRE_WARNING_NOTIFICATIONS)
+	case LTE_LC_EVT_TAU_PRE_WARNING:
+		if (!lwm2m_rd_client_ctx()) {
+			LOG_ERR("No lwm2m context");
```

Comment:
is this really an error?
How about if you intentionally shut down/close the client?
We could silently ignore it, or turn this into `LOG_DBG()`

Reply:
Good point, I changed the line to debug message